src/services/OutbreakService/OutbreakService.ts

```diff
@@ -212,6 +214,7 @@ export class OutbreakService {
   convertOutbreakEvents = (outbreakEvents: covidshield.OutbreakEvent[]): OutbreakEvent[] => {
     return outbreakEvents.map(event => {
       return {
+        id: getRandomString(8),
```

Comment:
Will add a comment here: "don't use this for anything besides the dedup code - it will change each time we get new data from the server"

Maybe rename to be: `dedupeId`